src/Visualizer/hooks.ts

```diff
@@ -82,12 +82,24 @@ export default function useHooks(
       reason: LayerSelectionReason | undefined,
       info: SelectedFeatureInfo | undefined,
     ) => {
-      if (selectedLayer.layerId === layerId && selectedLayer.featureId === featureId) return;
+      const isSketchLayer =
+        selectedLayer.layer?.layer.type === "simple" &&
+        selectedLayer.layer?.layer?.data?.isSketchLayer;
+      // Sketch layer feature has a fixed featureId, we need to exclude it from the skip condition
+      if (
+        selectedLayer.layerId === layerId &&
+        selectedLayer.featureId === featureId &&
+        !isSketchLayer
+      )
+        return;
 
       const computedLayer = await layer?.();
       const computedFeature =
         layerId && featureId
-          ? mapRef.current?.engine.findComputedFeatureById?.(layerId, featureId) ?? info?.feature
+          ? (isSketchLayer
+              ? computedLayer?.features?.find(f => f.id === featureId)
+              : mapRef.current?.engine.findComputedFeatureById?.(layerId, featureId)) ??
```

Comment:
The computed feature getting from engineAPI seems been cached, but i didn't finde where/how to change that, while getting the computedFeature from computedLayer data seems always updated, therefore i use this way when it's a sketch layer.

Reply:
I think `computedLayer?.features` doesn't have `properties`.
The `findComputedFeatureById` finds the feature from Cesium, and Cesium updates these feature in every frame, so you might need to wait one frame by requestAnimationFrame 🤔

Reply:
Emmm i don't know the reason but `computedLayer?.features` has `properties` and it's been used in Visualizer some places...
Maybe wait one frame doesn't help since i tested with update sketch layer several times, and the computedFeature's property is always the value when initialize...

Reply:
Ah for GeoJSON, it might be fine...